README.md

```diff
@@ -4,28 +4,25 @@
 [![LICENSE](https://img.shields.io/badge/license-apache2.0-green.svg)](https://github.com/kubernetes-sigs/kubefed/blob/master/LICENSE)
 [![Releases](https://img.shields.io/badge/version-v0.0.10-orange.svg)](https://github.com/kubernetes-sigs/kubefed/releases "Kubefed latest release")
 
-# Kubernetes Cluster Federation
-
-This repo contains an in-progress prototype of some of the
-foundational aspects of V2 of Kubernetes Federation.  The prototype
-builds on the sync controller (a.k.a. push reconciler) from
-[Federation v1](https://github.com/kubernetes/federation/) to iterate
-on the API concepts laid down in the [brainstorming
-doc](https://docs.google.com/document/d/159cQGlfgXo6O4WxXyWzjZiPoIuiHVl933B43xhmqPEE/edit#)
-and further refined in the [architecture
-doc](https://docs.google.com/document/d/1ihWETo-zE8U_QNuzw5ECxOWX0Df_2BVfO3lC4OesKRQ/edit#).
-Access to both documents is available to members of the
-[kubernetes-sig-multicluster google
-group](https://groups.google.com/forum/#!forum/kubernetes-sig-multicluster).
+# Kubernetes Cluster Federation (Kubefed)
+
+Kubernetes Cluster Federation (Kubefed for short) allows you to propagate
+resources to multiple Kubernetes clusters from a single set of APIs in a hosting
+cluster. Kubefed aims to provide essential mechanisms for expressing which
+clusters should be targetted and what resources should go where, with the aim of
```

Comment:
s/be targeted and what resources should go where/have their configuration managed and what that configuration should be/